From aefa138ddb6b8af8287b14b748e9943e5d9ddb2e Mon Sep 17 00:00:00 2001
From: Scott Murray <scott.murray@konsulko.com>
Date: Thu, 6 Oct 2022 11:56:38 -0400
Subject: [PATCH 1/3] Make Boost requirements more liberal

To allow building with Yocto Project 4.0/kirkstone or newer releases,
remove the EXACT specifier from the Boost find_package call.

Upstream-Status: Pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>
---
 kuksa-val-server/boost.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kuksa-val-server/boost.cmake b/kuksa-val-server/boost.cmake
index 54d7b2e..20a1e92 100644
--- a/kuksa-val-server/boost.cmake
+++ b/kuksa-val-server/boost.cmake
@@ -18,7 +18,7 @@ ADD_DEFINITIONS(-DBOOST_LOG_DYN_LINK)
 
 # Workaround function to allow cmake call `find_package` twice. Avoide side effects from local variables, which are produced be `find_package`
 function(findBoost Required)
-    find_package(Boost ${BOOST_VER} EXACT ${Required} 
+    find_package(Boost ${BOOST_VER} ${Required}
         COMPONENTS ${BOOST_COMPONENTS}
         OPTIONAL_COMPONENTS unit_test_framework
     )
-- 
2.37.3

